update build linux readme (#25274)
openvinotoolkit/openvino: docs/dev/build_linux.md

     mkdir build && cd build
   ```

> > **NOTE**: Disable oneAPI environment if has before this building process. 
> Please note that enabling the oneAPI environment during the compilation of OpenVINO from source code on Linux may cause the compilation to fail. It is recommended to disable the oneAPI environment during the build process to ensure successful compilation. Please proceed with caution and ensure to follow the build instructions accurately.  

Reviewer: ```suggestion
> **NOTE**: It is recommended to disable the oneAPI environment before compiling OpenVINO from source on Linux, as it may cause build failures. 
```
